aries-site/src/pages/components/layer/side-drawer-layer.mdx

```diff
@@ -0,0 +1,100 @@
+import { Notification } from 'grommet';
+import { Example } from '../../../layouts';
+import {
+  ConfigurationForm,
+  SideDrawerLayerAnatomy,
+  FilteringTable,
+  LayerStickyExample,
+  LayerStickyScrollExample,
+} from '../../../examples';
+
+## Anatomy
+
+This diagram highlights details specific to side drawer layers. See [general layer anatomy](/components/layer#anatomy) for specs that apply to all layers.
+
+<Example plain>
+  <SideDrawerLayerAnatomy />
+</Example>
+
+| Label | Region                   | Required?                                  | Purpose                                                           | Notes                                                                                                                             |
+| ----- | ------------------------ | ------------------------------------------ | ----------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| **1** | **Close button**         | Required on layers with no footer actions. | Closes the layer.                                                 | --                                                                                                                                |
```

Comment:
How does this work for screen reader users who open a side drawer layer, but want to exit without making changes? They have to navigate the entirety of the form to get to the cancel action?

I think we need to think through this side drawer scenario more. I don't think we can/should be dogmatic about the close and footer actions not coexisting in this scenario. I think this deserves some more nuanced considerations. 

For example, could the close button be backed up by a close confirmation modal if data is going to be lost? But in a filters context that would likely be overkill.

thinking.....